torch/onnx/symbolic_opset9.py

```diff
@@ -4298,7 +4298,9 @@ def linalg_vector_norm(g, self, ord, dim, keepdim, dtype):
     # Conditions based on https://pytorch.org/docs/stable/generated/torch.linalg.vector_norm.html
```

Comment:
Could you help us annotate the function parameter types to make them clear too? Thanks a lot!

Reply:
No prob.